modal/object.py

```diff
@@ -215,7 +217,7 @@ async def resolve(self):
                 " wasn't defined in global scope."
             )
         else:
-            resolver = Resolver()
+            resolver = Resolver(client=self._client)
```

Comment:
Looking at the args for `Resolver`, should we also supply `environment_name` here? Not 100% sure how that works

Reply:
Hm good question. It might not matter but let me think it through.